common/Geometry2d/Point.hpp

```diff
@@ -51,25 +64,32 @@ class Point {
      * does vector addition
      * adds the + operator, shorthand
      */
-    Point operator+(const Point& other) const {
+    Point operator+(Point other) const {
         return Point(x + other.x, y + other.y);
     }
 
     /**
      * see operator+
      * does vector division, note the operator
-     * without parameter, it is the negative
      */
-    Point operator/(const Point& other) const {
+    Point operator/(Point other) const {
         return Point(x / other.x, y / other.y);
     }
 
     /**
      * see operator+
+     * does vector multiplication, note the operator
+     */
+    Point operator*(Point other) const {
```

Comment:
This one might need a little more explanation since it's not the dot or cross product.  It's sort of an unusual operation to perform.